simplemind/__init__.py

```diff
@@ -77,7 +77,12 @@ def create_conversation(
 
 
 def generate_data(
-    prompt, *, llm_model=None, llm_provider=None, response_model=None, **kwargs
+    prompt: str,
+    *,
+    llm_model: str | None = None,
```

Comment:
why is this better than Optional?

Reply:
While they are the same thing, I guess it is better to explicitly state the thing instead of "implying" it via Optional

Reply:
ah interesting. not opposed